scripts/jenkins/jobs-ibs/cloud-mkcloud6-suse-trigger-new.yaml

```diff
@@ -0,0 +1,61 @@
+- job:
+    name: 'cloud-mkcloud6-suse-trigger-new'
+    project-type: multijob
+    node: cloud-trigger
+
+    triggers:
+      - pollurl:
+          cron: 'H/5 * * * *'
+          polling-node: cloud-trigger
+          urls:
+            - url: 'http://clouddata.cloud.suse.de/cgi-bin/grep/download.suse.de/ibs/SUSE:/SLE-12-SP1:/Update:/Products:/Cloud6/images/iso/?regexp=x86_64'
+              check-content:
+                - simple: true
+
+    logrotate:
+      numToKeep: 7
+      daysToKeep: -1
+
+    builders:
+      - multijob:
+          name: Standard Gate Checks (non-HA/OVS+Linuxbridge)
+          condition: SUCCESSFUL
+          projects:
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                cloudsource=susecloud6
+                TESTHEAD=
+                nodenumber=2
+                mkcloudtarget=all_batch
+                scenario=cloud6-2nodes-default.yml
+                want_node_aliases=controller=1,compute-kvm=1
+            - name: openstack-mkcloud
+              node-label: cloud-trigger
+              predefined-parameters: |
+                cloudsource=susecloud6
+                TESTHEAD=
+                nodenumber=4
+                want_ceph=1
+                networkingplugin=linuxbridge
+                mkcloudtarget=all
```

Comment:
Would it make sense to create real triggers for these jobs (instead of defining the parameters here)?

(can be done later, since it's been like this forever -- just wondering out loud)